lib/matplotlib/tri/trirefine.py

```diff
@@ -64,30 +63,29 @@ def __init__(self, triangulation):
 
     def refine_triangulation(self, return_tri_index=False, subdiv=3):
         """
-        Computes an uniformly refined triangulation *refi_triangulation* of
+        Compute an uniformly refined triangulation *refi_triangulation* of
         the encapsulated :attr:`triangulation`.
 
         This function refines the encapsulated triangulation by splitting each
         father triangle into 4 child sub-triangles built on the edges midside
-        nodes, recursively (level of recursion *subdiv*).
-        In the end, each triangle is hence divided into ``4**subdiv``
-        child triangles.
-        The default value for *subdiv* is 3 resulting in 64 refined
-        subtriangles for each triangle of the initial triangulation.
+        nodes, recursing *subdiv* times.  In the end, each triangle is hence
+        divided into ``4**subdiv`` child triangles.
 
         Parameters
         ----------
         return_tri_index : bool, default: False
             Whether an index table indicating the father triangle index of each
-            point will be returned.
+            point is returned.
         subdiv : int, default: 3
             Recursion level for the subdivision.
-            Each triangle will be divided into ``4**subdiv`` child triangles.
+            Each triangle is divided into ``4**subdiv`` child triangles;
+            hence, the default results in 64 refined subtriangles for each
+            triangle of the initial triangulation.
 
         Returns
         -------
-        refi_triangulation : :class:`~matplotlib.tri.Triangulation`
-            The returned refined triangulation
+        refi_triangulation : `.Triangulation`
```

Comment:
```suggestion
        refi_triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above